src/vm/eventpipe.cpp

```diff
@@ -29,6 +32,7 @@ void EventPipe::Initialize()
         (CrstFlags)(CRST_TAKEN_DURING_SHUTDOWN));
 
     s_pConfig = new EventPipeConfiguration();
+    InitProvidersAndEvents();
```

Comment:
Can you put a comment here that says that this calls into auto-generated code to initialize the runtime providers and events so that the EventPipe configuration lock isn't taken at runtime.